docs/installation.md

```diff
@@ -63,6 +63,22 @@ BENTOV2_AUTH_REALM=bentov2
 BENTOV2_AUTH_WELLKNOWN_PATH=/auth/realms/${BENTOV2_AUTH_REALM}/.well-known/openid-configuration
 ```
 
+If an *external* keycloak instance is being used and you would like to set it up using `./bentoctl.bash init-auth`, follow the steps above for external and then below:
+- Set up a keycloak instance and provide its url in `BENTOV2_AUTH_PUBLIC_URL`
+- Make the desired realm for bento, update `BENTOV2_AUTH_REALM` accordingly
+- Create an administrative user account in the Bento realm
+- In _Realm Roles_, create an admin role, which we will give to the user later
+- In the admin role, go to the associated roles tab and assign all the **realm-management** roles to the admin realm role.
+- In the admin user page, select the _Role Mapping_ tab and click on the _Assign Role_ button
+- Select _Filter By Realm Roles_, select the admin role and click _Assign_
+- The user account can then be used to remotely configure the realm with the admin API
+- Setup the following variables accordingly `BENTOV2_AUTH_ADMIN_USER` & `BENTOV2_AUTH_ADMIN_PASSWORD`
+- Set `BENTOV2_USE_EXTERNAL_KEYCLOAK` in local.env as 1
+
+> This administrative user account should only be used to configure a Keycloak realm for a Bento instance.
+> As such, once your Bento instance is properly configured with your remote Keycloak, we recommend that you unasign the admin realm role from the admin user and reset his credentials. Doing so mitigates the potential security risks caused by leaked admin credentials.
```

Comment:
shouldn't we just suggest they delete the account? if we keep this, typos: unasign -> unassign; don't use a gendered pronoun

Reply:
It is upto them I guess ,as the above isn't bad either.